src/main/java/carpet/helpers/HopperCounter.java

```diff
@@ -67,7 +67,7 @@
         {
             counterMap.put(color, new HopperCounter(color));
         }
-        COUNTERS = Maps.immutableEnumMap(counterMap);
+        COUNTERS = Collections.unmodifiableMap(counterMap);
```

Comment:
Isn't enum map faster or the collections method is smart enough to figure it out?

Reply:
`Collections.unmodifiableMap` is the same as `Maps.immutableEnumMap`: Just a wrapper over the other map. Since it's already an EnumMap, it should have the same performance as the guava version, because they both do the same.

Only difference is that guava's method creates a copy of the map instead of saving the same one, and has a custom class for it. Since the old map is not passed anywhere, there's no problem with that.